automatic/awscli/awscli.nuspec

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Do not remove this test for UTF-8: if “Ω” doesn’t appear as greek uppercase omega letter enclosed in quotation marks, you should use an editor that supports UTF-8, not this one. -->
+<package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
+  <metadata>
+    <id>awscli</id>
+    <version>1.18.4</version>
+    <packageSourceUrl>https://github.com/chocolatey-community/chocolatey-coreteampackages/tree/master/automatic/awscli</packageSourceUrl>
+    <owners>chocolatey, LudicrousByte, dtgm</owners>
+    <title>AWS Command Line Interface (Install)</title>
+    <authors>Amazon Web Services</authors>
+    <projectUrl>https://aws.amazon.com/cli/</projectUrl>
+    <iconUrl>https://rawcdn.githack.com/jtcmedia/chocolatey-packages/464457cc57f33f6243c27246755e4ac08ca3a9fb/icons/awscli.png</iconUrl>
```

Comment:
wrong url

Reply:
Not important IMO, as it will be updated automatically by one of the helpers in the repository.